proposals/lip-define_new_transaction_schema.md

```diff
@@ -0,0 +1,189 @@
+
+This LIP is licensed under the [Creative Commons Zero 1.0 Universal](https://creativecommons.org/publicdomain/zero/1.0/).
+
+
+## Motivation
+
+The Lisk protocol handles identifiers for transactions, modules, commands, and many more. The type of those identifiers is however not fully consistent, as some are of type `uint32` (like module ID and chain ID) and others of type `bytes` (like transaction ID and block ID). Moreover, all identifiers used in the new state model to compute the store keys must be first converted to type `bytes`. Unifying all identifier types to be of type `bytes` simplifies their handling and avoids unnecessary type conversion.
+
+
+## Rationale
+
+
+### Unifying Identifier Type
+
+The type identifiers in the Lisk protocol are not fully consistent, as some are of type `uint32` and others of type `bytes`. This means that special care is required to use the proper type whenever the identifier is used. Further, identifiers are often used to compute the store keys in the state tree and for this purpose must always be of type bytes. Hence, identifiers of `uint32` type need to be converted to fulfill this purpose. This implies that the implementation very often converts those identifiers from their integer form (as schema entries) to the corresponding bytes (as store key). The new transaction schema introduced in this LIP sets the module ID and command ID to type bytes. 
+
+Defining identifiers as type `bytes` also requires fixing the length of the identifier. This was not possible when using identifiers of type `uint32`, as the full 4 bytes of the maximal range always had to be assumed when using the identifier in the state tree.  
+
+
+### New Property Names
+
+All properties in the proposed transaction schema are equivalent to the ones defined in LIP 0028. The only changes are the renaming of `assetID` to `commandID` and of `asset` to `params`. As was described in LIP "Update Lisk SDK modular blockchain architecture".
+
+
+## Specification
+
+The transaction schema defined in [LIP 0028](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0028.md) is superseded by the one defined below. 
+
+The `params` property must follow the schema corresponding to the `moduleID`, `commandID` pair defined in the corresponding module.
+
+All transaction procedures - serialization, deserialization, signature calculation, signature validation and transaction ID - follow the same specifications already defined in [LIP 0028](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0028.md#serialization). The resulting serialization or signatures are however different when the proposed transaction schema is used.
+
+
+### Constants
+
+
+| Global Constants                |         |                                                                                                            |
+|:-------------------------------:|:-------:|------------------------------------------------------------------------------------------------------------|
```

Comment:
I think the colons (:) are centering  the entries in the table, I would remove them

Reply:
Usually the "name", "value", "description" raw is centered, but couldn't make it here due to the "global constants" raw. Not a big deal, could change it if you insist :-)

Reply:
I changed the table and there is no centering anymore ;-)